aexpect/remote_door.py

```diff
@@ -174,13 +174,20 @@ def run_remotely(function):
     and is expected to follow PEP8 standards.
     """
     def wrapper(session, *args, **kwargs):
+        # drop first argument and first line with the decorator since function already runs remotely
+        fn_source = inspect.getsourcelines(function)[0][1:]
+        indent_number = len(fn_source[0]) - len(fn_source[0].lstrip())
+
+        # remove extra indentation from the beginning to allow indented functions
+        if indent_number > 0:
+            fn_source = [line[indent_number:] for line in fn_source if len(line) > indent_number]
```

Comment:
Do you have a valid reason for the `if len(line) > indent_number`? If it's not longer than not including it or including `line[indent_number:]` should result in the same (``). (not a strict thing just wondering)

Reply:
Good point, I know that it is normal in `numpy` but wasn't sure here. It works fine as you suggested.